.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v3
       with:
-        go-version: ^1.18
+        go-version: '^1.19.0'

```

Comment:
The `.0` addition should allow patch version updates but prevent CI from bumping itself up a minor version without us explicitly changing this file.

Reply:
It looks like the latest version is [go1.19.3](https://github.com/golang/go/releases/tag/go1.19.3).
If you don't have any reason, Could you consider using the latest minor version?

Reply:
The `^` should make that happen automatically, and also automatically pick up new patch-level versions as they are available. Currently it is picking up 1.19.2:

![image](https://user-images.githubusercontent.com/4789493/202337452-f89d01d3-f615-48c3-8fb0-18c9fa3bf838.png)

Reply:
Oh, Sorry. 
I didn't know that. Thank you for showing the details!